src/test/frank-single-transaction.sh

```diff
@@ -33,5 +33,5 @@ FDDEV_PID=$!
 sleep 4
 sudo nsenter --net=/var/run/netns/veth_test_xdp_1 ${TEST_QUIC_TXN}
 RETVAL=$?
-sudo kill -9 $FDDEV_PID
+sudo kill $FDDEV_PID
```

Comment:
Why remove the `-9`? I vaguely recall a scenario in the past where `fddev` could fail in a way that requires `-9`.